.golangci.yml

```diff
@@ -153,10 +153,8 @@ linters:
   enable:
     - revive
     - goimports
-    - varcheck
+    - unused
     - unparam
-    - deadcode
-    - structcheck
     - errcheck
```

Comment:
What was the problem with these deleted module?

Reply:
deprecated by new version; replaced with unused